docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -176,3 +176,32 @@ For the remaining users:
 
 .. literalinclude:: resource-groups-example.json
     :language: json
+
+.. _scheduleweight-ex:
+
+``schedulingWeight`` example
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For each resource subgroup, the weighted scheduling is essentially a raffle
+ticket. If your ``schedulingWeight`` is 10, you’ve bought 10 tickets, and the
+other subgroup whose weight is 1 only bought 1. All of the tickets get thrown
+into the same jar, and Trino picks one ticket at random every time it has
+capacity to run a query. 
+
+For example, if you had three subgroups under global defined like the
+example below you would end up with this type of distribution:
+
+.. code-block:: text
+
+    data_definition - schedulingWeight: 1 - share: 8.3% (1/12)
+    adhoc - schedulingWeight: 10 - share: 83.3% (10/12)
+    pipeline - schedulingWeight: 1 - share: 8.3% (1/12)
```

Comment:
This seems backwards - setting pipelines up to fail in favor of random queries. Let's ping @waraposn to see what he says.

Reply:
In this example, adhoc/interactive queries are given priority over pipelines/batch workload -- I think adding something along the lines of "the above example prioritizes adhoc/interactive queries over pipelines/batch workload by setting a higher scheduling weight for `adhoc`" will help someone who's new at configuring resource groups

Reply:
Maybe adding another example for giving pipelines higher priority would be helpful as well or a split that is more balanced.

Reply:
@waraposn what do you think of what @m57lyra helped me put together? Does that work or did you have something else in mind?